src/Components/SearchBox.jsx

```diff
@@ -0,0 +1,29 @@
+const SearchBox = ({ songList, selectedSongTitle, setSelectedSongTitle }) => {
+  return (
+    <>
+      <section className="search-container">
+        <div className="searchBox wrapper">
+          <h2>Please select a song below:</h2>
+          <form className="searchForm" action="submit">
+            <select
+              value={selectedSongTitle}
+              onChange={(e) => {
+                setSelectedSongTitle(e.target.value);
+              }}
+            >
+              <option value="">Please select a song</option>
```

Comment:
consider adding after your value a disable function 
<option value="
----> add that ---> " disabled="disabled"
> 
to disable user from selecting "Please select a song"

Reply:
Done!